diff --git a/src/config.c b/src/config.c
--- a/src/config.c
+++ b/src/config.c
@@ -83,9 +83,9 @@ int32 JoystickAnalogSaturate[MAXJOYAXES];
 //
 int32 ScreenMode = 1;
 int32 ScreenDisplay = 0;
-int32 ScreenWidth = 640;
-int32 ScreenHeight = 480;
-int32 ScreenBPP = 8;
+int32 ScreenWidth = 1280;
+int32 ScreenHeight = 720;
+int32 ScreenBPP = 32;
 int32 ForceSetup = 1;
 
 extern char WangBangMacro[10][64];
@@ -202,9 +202,9 @@ void CONFIG_SetDefaults( void )
 
    ScreenMode = 1;
    ScreenDisplay = 0;
-   ScreenWidth = 640;
-   ScreenHeight = 480;
-   ScreenBPP = 8;
+   ScreenWidth = 1280;
+   ScreenHeight = 720;
+   ScreenBPP = 32;
    FXDevice = 0;
    MusicDevice = 0;
    NumVoices = 32;
@@ -227,9 +227,9 @@ void CONFIG_SetDefaults( void )
    Bstrcpy(WangBangMacro[8], MACRO9);
    Bstrcpy(WangBangMacro[9], MACRO10);
 
-   CONFIG_SetDefaultKeyDefinitions(CONFIG_DEFAULTS_CLASSIC);
-   CONFIG_SetMouseDefaults(CONFIG_DEFAULTS_CLASSIC);
-   CONFIG_SetJoystickDefaults(CONFIG_DEFAULTS_CLASSIC);
+   CONFIG_SetDefaultKeyDefinitions(CONFIG_DEFAULTS_MODERN);
+   CONFIG_SetMouseDefaults(CONFIG_DEFAULTS_MODERN);
+   CONFIG_SetJoystickDefaults(CONFIG_DEFAULTS_MODERN);
 
    memset(MouseDigitalAxes, -1, sizeof(MouseDigitalAxes));
    for (i=0; i<MAXMOUSEAXES; i++) {
