diff --git a/src/config.c b/src/config.c
index 98df57e..2d4ec58 100644
--- a/src/config.c
+++ b/src/config.c
@@ -37,8 +37,8 @@ int32 SetupSound;
 int32 SetupMusic;
 int32 FxOn;
 int32 MusicOn;
-int32 ScreenWidth = 1280;
-int32 ScreenHeight = 720;
+int32 ScreenWidth = 1920;
+int32 ScreenHeight = 1080;
 int32 ScreenBPP = 32;
 int32 ScreenMode;
 int32 ScreenDisplay;
@@ -572,8 +572,8 @@ void CONFIG_SetDefaults(void)
 	SetupSound = 1;
 	SetupMusic = 1;
 	FxOn = 1;
 	MusicOn = 1;
-	ScreenWidth = 1280;
-	ScreenHeight = 720;
+	ScreenWidth = 1920;
+	ScreenHeight = 1080;
 	ScreenBPP = 32;
 	ScreenMode = 0;
 	ScreenDisplay = 0;
